modules/cli-logging-in-web.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_cli/getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="cli-logging-in-web_{context}"]
+= Logging in to the OpenShift CLI using a web browser
+
+You can log in to the OpenShift CLI (`oc`) with the help of a web browser to access and manage your cluster. This allows users to avoid inserting their access token into the command line.
+
+[WARNING]
+====
+Logging in through the web browser runs a server on localhost with HTTP, not HTTPS. Use with caution on multi-user workstations.
```

Comment:
```suggestion
On multi-user workstations, logging in through the web browser runs a server on localhost with HTTP, not HTTPS. HTTP should be used with caution. 
```

Reply:
This suggestion changes the meaning a bit, and I don't think it's quite accurate. It's not only on multi-user workstations that the server runs with HTTP.

Reply:
Is that better?

Reply:
I think there could be some room for improvement here, but I still think that changes the meaning and is less clear

Reply:
I think I would keep it as two sentences. The first explains what happens. And then the second is cautionary.

I might make the following suggestion (to be clearer it's about logging into the CLI w/ a browser, not logging into the web console w/ a browser). And we could join them with a semicolon. But I think we have to separate the statement about it using HTTP and using caution on multi-user workstations

```suggestion
Logging in to the CLI through the web browser runs a server on localhost with HTTP, not HTTPS; use with caution on multi-user workstations.
```

Reply:
Change made, thank you!